src/Contact/ContactController.php

```diff
@@ -30,11 +30,6 @@ public function __construct(
         parent::__construct($responseFactory, $user, $aliases, $view);
     }
 
-    protected function getId(): string
-    {
-        return 'contact';
```

Comment:
Wouldn't new one result in `contact/contact`?

Reply:
No
![image](https://user-images.githubusercontent.com/6815714/85921901-43c0e700-b888-11ea-92fd-70838ac501ce.png)